packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js

```diff
@@ -0,0 +1,16 @@
+export const TRUSTED_ZONE_CALLBACK_PHONE_PAGES = {
```

Comment:
**question:** Is there any reason to name it this way? Why Phone?
cc @mohammed-zahaf